config/HELICSConfig.cmake.in

```diff
@@ -47,37 +47,41 @@ find_program(HELICS_BROKER helics_broker
 	PATH ${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_BINDIR@
 	NO_DEFAULT_PATH)
 	
-set(HELICS_HAVE_ZEROMQ @HELICS_HAVE_ZEROMQ@)
-set(HELICS_HAVE_MPI @HELICS_HAVE_MPI@)
+set(ENABLE_ZMQ_CORE @ENABLE_ZMQ_CORE@)
+set(ENABLE_MPI_CORE @ENABLE_MPI_CORE@)
```

Comment:
should the rest of the ENABLE_XXX_CORE be here two or are these here for the library linking?

Reply:
These are for the library linking.

Reply:
@phlptp So these variables are set in the context of the CMake project calling find_package. Do you think we should set variables for the rest of the core types (HELICS_HAVE_XXX_CORE), or unset these two variables before the script exits?

Reply:
You can query individual core types using isCoreTypeAvailable functions.    It might not be a bad idea to have some function that prints available core types I suppose.  But I agree there is not much use for a configure time variable for applications using HELICS.  So I would be inclined to have them be unset after we are done with them.

Reply:
So I think if we add some code to unset these variables when we are done with them, this is ready to go?

Reply:
And once the MSVC change is also in. I think it should be possible to get rid of the variables here so they don't need to be unset.